agir/front/components/toolsPage/index.js

```diff
@@ -0,0 +1,470 @@
+    align-items: center;
+    display: flex;
+
+    justify-content: space-between;
+    flex-grow: 1;
+    padding: 20px;
+  }
+`;
+
+const BannerToolContainer = styled.div`
+  width: 100%;
+  background-color: ${style.primary500};
+  color: white;
+  font-size: 48px;
+  font-weight: 700;
+  padding: 42px;
+  height: 138px;
+  overflow: hidden;
+  position: relative;
+  border-radius: 0.5rem;
+
+  div {
+    top: 60px;
+    right: -64px;
+    width: 600px;
+    height: 170px;
+    position: absolute;
+    background-color: ${style.redNSP};
+    transform: rotate(-9deg);
+  }
+`;
+
+const BannerHelpContainer = styled.div`
+  width: 100%;
+  background-color: ${style.secondary100};
+  font-size: 20px;
+  font-weight: 700;
+  overflow: hidden;
+  position: relative;
+  border-radius: 0.5rem;
+  padding: 47px;
+
+  @media (max-width: ${style.collapse}px) {
+    padding: 24px;
+  }
+
+  div {
+    @media (max-width: ${style.collapse}px) {
+      display: none;
+    }
+    bottom: -38px;
+    right: 0;
+    width: 530px;
+    height: 100%;
+    position: absolute;
+    background-image: url(${bricksNSP});
+    background-repeat: no-repeat;
+    // transform: rotate(-9deg);
+    transform: scaleX(-1);
+  }
+`;
+
+const WEBSITES = [
+  {
+    title: "Mélenchon 2022",
+    img: img_JLM_2022,
+    href: nonReactRoutes.jlm2022,
+  },
+  {
+    title: "L'avenir en commun",
+    img: img_AvenirEnCommun,
+    href: nonReactRoutes.programme,
+  },
+  {
+    title: "L'insoumission",
+    img: img_Linsoumission,
+    href: nonReactRoutes.linsoumission,
+  },
+  {
+    title: "Comparateur de programme",
+    img: img_Comparateur,
+    href: nonReactRoutes.comparateur,
+  },
+];
+
+const LinkInfoAction = () => (
+  <StyledButton
+    small
+    as="Link"
+    color="secondary"
+    href="https://materiel.lafranceinsoumise.fr/"
+    target="_blank"
+  >
+    Accéder au fiches pratiques
+    <RawFeatherIcon
+      name="arrow-up-right"
+      color={style.black1000}
+      width="1.25rem"
+    />
+  </StyledButton>
+);
+
+const LinkMaterial = () => (
+  <StyledButton
+    small
+    as="Link"
+    color="secondary"
+    href="https://materiel.lafranceinsoumise.fr/"
+    target="_blank"
+  >
+    Accéder au site matériel
+    <RawFeatherIcon
+      name="arrow-up-right"
+      color={style.black1000}
+      width="1.25rem"
+    />
+  </StyledButton>
+);
+
+const BannerTool = () => (
+  <BannerToolContainer>
+    Outils
+    <div />
+  </BannerToolContainer>
+);
+
+const BannerHelp = () => (
+  <BannerHelpContainer>
+    Une question&nbsp;? Un&nbsp;problème&nbsp;? Pas&nbsp;de&nbsp;panique&nbsp;!
+    <br />
+    <Spacer size="30px" />
+    <StyledButton
+      small
+      as="Link"
+      color="secondary"
+      href="https://infos.actionpopulaire.fr"
+      target="_blank"
+    >
+      Accéder au centre d'aide
+      <RawFeatherIcon
+        name="arrow-up-right"
+        color={style.black1000}
+        width="1.25rem"
+      />
+    </StyledButton>
+    <div />
```

Comment:
Tu peux peut-être faire ça directement dans les styles avec un `::after`, plutôt qu'utiliser une div vide ?

Reply:
J'ai réussi à remplacer avec un `::after` pour le `BannerHelp` qui contient une image mais pas pour le `BannerTool` qui contient qu'un background-color. Si tu vois ce qui bloque je suis preneur !

Reply:
Ça devrait fonctionner comme ça : 
```js
const BannerToolContainer = styled.div`
  width: 100%;
  height: 138px;
  background-color: ${style.primary500};
  color: white;
  font-size: 48px;
  font-weight: 700;
  padding: 42px;
  margin-bottom: 48px;
  overflow: hidden;
  position: relative;
  border-radius: ${(props) => props.theme.borderRadius};

  &::after {
    content: "";
    display: block;
    top: 60px;
    right: -64px;
    width: 600px;
    height: 170px;
    position: absolute;
    background-color: ${style.redNSP};
    transform: rotate(-9deg);
  }
`;

const BannerHelpContainer = styled.div`
  width: 100%;
  background-color: ${style.secondary100};
  font-size: 20px;
  font-weight: 700;
  overflow: hidden;
  position: relative;
  border-radius: ${(props) => props.theme.borderRadius};
  padding: 47px;

  @media (max-width: ${style.collapse}px) {
    padding: 24px;
  }

  &::after {
    content: "";
    display: block;
    bottom: -38px;
    right: 0;
    width: 530px;
    height: 100%;
    position: absolute;
    transform: scaleX(-1);
    background-image: url(${bricksNSP});
    background-repeat: no-repeat;

    @media (max-width: ${style.collapse}px) {
      display: none;
    }
  }
`;
```
Pour que les ::before and ::after s'affichent il faut toujours spécifier une propriété `content`, même si la valeur est une chaîne de caractères vide.

Reply:
Oui ça marchait pour le `BannerHelp` avec l'image et `content: url(${bricksNSP});`
Mais j'ai du faire une fausse manip avec le `BannerTool` car en effet là ça marche avec le content vide, merci ! :)